tests/test_precompiled.py

```diff
@@ -593,13 +593,27 @@ def test_redact_precompiled_letter_address_block_sends_log_message_if_multiple_m
     assert "More than one match for address block during redaction procedure" in str(exc_info.value)
 
 
-def test_escape_special_characters_for_regex_matches_string():
-    string = 'PEA NUTT + MIKO JELLY4 JELLY COURT (PEANUT BUTTER JELLY WHARF)TOAST STR.ALLDAYSNACKSHIRESNACKISTANSN1 PBJ'
+@pytest.mark.parametrize('string', [
```

Comment:
Good work on the test cases with nice comments

Reply:
thanks :)